packages/test/src/activities/index.ts

```diff
@@ -92,3 +93,16 @@ export async function progressiveSleep(): Promise<void> {
   await sleep(100);
   Context.current().heartbeat(3);
 }
+
+export async function queryOwnWf(queryDef: QueryDefinition<any>): Promise<void> {
```

Comment:
If you're adding this, why not make it:
```suggestion
export async function queryOwnWf<R, A extends any[]>(queryDef: QueryDefinition<R, A>, ...args: A): Promise<R> {
```